apps/code-examples/src/app/code-examples/pages/page/home-page-blocks-layout-demo/demo.component.html

```diff
@@ -0,0 +1,27 @@
+<sky-page layout="blocks">
+  <sky-page-header pageTitle="Home">
+    <sky-page-header-actions>
+      <button type="button" class="sky-btn sky-btn-default">
+        <sky-icon icon="add" /> New
+      </button>
+    </sky-page-header-actions></sky-page-header
```

Comment:
Possible to add a newline after `</sky-page-header-actions>`?

Reply:
Good call. Updated.